website/templates/project/settings.mako

```diff
@@ -270,7 +271,74 @@
 
         % endif  ## End Configure Commenting
 
+        % if 'admin' in user['permissions'] or (discussions['enabled'] and user['has_read_permissions']): ## Begin Configure Discussions
+
+            % if not node['is_registration']:
+
+                <div class="panel panel-default">
+                    <span id="configureDiscussionsAnchor" class="anchor"></span>
+
+                    <div class="panel-heading clearfix">
+                        <h3 class="panel-title">Configure Email Discussions</h3>
+                    </div>
+
+                    % if discussions['enabled']:
+                        <div class="help-block" style="padding-left: 15px">
+                            <p>Start an email discussions by sending an email to <a href="mailto: ${node['id']}@osf.io">${node['id']}@osf.io</a>.</p>
```

Comment:
This should probably use some sort of variable for the domain so that it works properly on staging. Perhaps a new property for the Node class, in which case there are some tests that could be made refactored as well.